File: examples/rc-car-send/code.js
radio.setGroup(1)
basic.forever(function on_forever() {
    if (input.buttonIsPressed(Button.AB)) {
        radio.sendNumber(1)
    } else if (input.buttonIsPressed(Button.A)) {
        radio.sendNumber(2)
    } else if (input.buttonIsPressed(Button.B)) {
        radio.sendNumber(3)
    } else {
        radio.sendNumber(0)
    }
    
})

File: examples/rc-car-send/code.py
radio.set_group(1)

def on_forever():
    if input.button_is_pressed(Button.AB):
        radio.send_number(1)
    elif input.button_is_pressed(Button.A):
        radio.send_number(2)
    elif input.button_is_pressed(Button.B):
        radio.send_number(3)
    else:
        radio.send_number(0)
basic.forever(on_forever)
